packages/embark/src/cmd/dashboard/repl.js

```diff
@@ -19,9 +19,10 @@ class REPL {
     }
   }
 
-  enhancedEval(cmd, _context, _filename, _callback) {
+  enhancedEval(cmd, _context, _filename, callback) {
     this.events.request('console:executeCmd', cmd.trim(), (err, message) => {
       this.logger.info(message === undefined ? '' : message);
```

Comment:
Why must we print the message via logger, as opposed to passing the object/message to the `REPL` and letting it handle it's display (as it was before). I understand that we want to print the message in the non-console dashboard, but couldn't that be handled by only logging when detecting an IPC connection (ie `ipc.isServer() && ipc.connected`).

Reply:
It's just that with that way, the logger is the only source of logging. Which handles transmitting to cockpit etc